### ISSUE TITLE
Welp, we have a big problem.

### ISSUE THREAD

**Issue by tokotmg** [weespin/WillFromAfarDownloader#47]
![Capture](https://github.com/user-attachments/assets/dbd04e7d-cdf3-4064-85b8-adf112ea1609)

**Comment by levi-creator-max** [comment]
What the! But API died, we can't do Acapela Downloader like this! Waleed, API is gone! Forever! [crying]!

**Comment by levi-creator-max** [comment]
I hate this mess!

**Comment by weespin** [comment]
@tokotmg 
Yep, sadly the Tap and Talk Exploit is not working anymore as they've removed the last API part from the Android Auth process.
I'm going to archive this project indefinitely.
